tests/conftest.py

```diff
@@ -0,0 +1,12 @@
+import os, pytest
+from flaskr import create_app
```

Comment:
PyTest fails here since it cannot find module flaskr.

In the tutorial, the flaskr app is installed first where it can be imported in any directory, including /tests/. If you got it to work, feel free to override this comment.

**EDIT** I found a workaround: `python -m pytest`